mplc/dataset.py

```diff
@@ -189,8 +189,8 @@ def generate_new_model(self):
         model.add(Dense(self.num_classes))
         model.add(Activation('softmax'))
 
-        # initiate RMSprop optimizer
-        opt = RMSprop(learning_rate=0.0001, decay=1e-6)
+        # initiate Adam optimizer
+        opt = Adam(learning_rate=0.001, decay=1e-5)
```

Comment:
https://keras.io/api/optimizers/adam/
what is decay ? it is the beta_1 in the doc ?

Reply:
No, beta_1 and beta_2 are for the momentum. 
As far as I understood the doc, the decay is for the learning rate.